src/main/java/wolox/training/exceptions/BookIdMismatchException.java

```diff
@@ -0,0 +1,13 @@
+package wolox.training.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import wolox.training.constants.ExceptionConstants;
+
+@ResponseStatus(code = HttpStatus.NOT_ACCEPTABLE, reason = ExceptionConstants.BOOK_ID_MISMATCH)
+public class BookIdMismatchException extends RuntimeException {
+
+  public BookIdMismatchException() {
+    super(ExceptionConstants.BOOK_ID_MISMATCH, new Exception());
```

Comment:
I think you can either:
a) use `this` instead of a new Exception in this constructor
b) or use `super(ExceptionConstants.BOOK_ID_MISMATCH);` (I believe there is that constructor too)